lib/logger.js

```diff
@@ -257,14 +266,9 @@ class Logger extends EventEmitter {
       this.env = options.env
     }
 
-    if (has(options, 'with_credentials')) {
-      console.warn(
-        '[Deprecated] with_credentials is deprecated.  Please use withCredentials.'
-      )
-      options.withCredentials = options.with_credentials
-    }
-    if (has(options, 'withCredentials')) {
-      withCredentials = Boolean(options.withCredentials)
+    if (has(options, 'withCredentials') || has(options, 'with_credentials')) {
```

Comment:
@smusali can you or @jakedipity double check me here?  I see no reason to even have CORS type things for this client.  It's just a POSTing client.  In any case, this option is not a direct option of `got`, so it's coming out.

Reply:
I can't immediately see how `got` does CORS requests.s. I'm sure this is needed for when this is bundled for the FE, but we need to test it so we don't lose functionality.

https://github.com/logdna/nodejs/pull/36